lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -347,357 +280,387 @@
         }
       }
     }
+  }
+}
 
-    // ====================
+// +Dashboard - Course
+// ==================== 
+.dashboard .my-courses {
+
+  // UI: individual course item
+  .course {
+    @include box-sizing(box);
+    @include transition(all 0.15s linear 0s);
+    @include clearfix();
+    @extend %ui-depth2;
+    position: relative;
+    margin: ($baseline/2);
 
-    // UI: individual course item
-    .course {
-      @include box-sizing(box);
-      @include transition(all 0.15s linear 0s);
+    .details {
       @include clearfix();
-      @extend %ui-depth2;
-      position: relative;
 
       .cover {
         @include box-sizing(border-box);
         @include transition(all 0.15s linear 0s);
+        @include float(left);
+        @include margin-right(flex-gutter());
+        width: flex-grid(3);
         overflow: hidden;
+        display: block;
         position: relative;
-        @include float(left);
-        height: 100%;
         max-height: 100%;
-        width: 200px;
         height: 120px;
-        margin: 0;
-        border-radius: ($baseline/10);
+        border-radius: ($baseline/5);
         border: 1px solid $dashboard-course-cover-border;
         border-bottom: 4px solid $dashboard-course-cover-border;
-        padding: ($baseline/10);
+
+        &:before {
```

Comment:
I would prefer that you wrap the image in a div and float that rather than force the link space with this :before.